### PR TITLE
 EDSC-2134: Create clean branch as starting point for EDSC 2.0

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,206 +0,0 @@
-# Making Changes
-
```
Comment (trevorlang) [inline CONTRIBUTING.md]: I feel like we should keep this and update, no?

File: static/src/js/components/Header/Header.js [hunk 2]
```diff
@@ -1,101 +1,34 @@
...
+const Header = () => (
+  <header className="header">
+    <ul className="header__menu">
+      <li className="header__menu-item">
+        <h1 className="header__menu-title">
+          <Link
+            className="header__menu-link"
+            to="/search"
+          >
+              CSW User Interface
```
Comment (trevorlang) [inline static/src/js/components/Header/Header.js]: Leftovers :)

File: static/src/js/containers/SidebarContainer/Header/Header.scss [hunk 3]
```diff
@@ -6,22 +6,22 @@
 .header__site-logo {
   display: flex;
   align-items: flex-end;
-  background-image: url('../../../../../assets/images/app-logo.png');
+  background-image: url('../../../../assets/images/app-logo.png');
```
Comment (trevorlang) [inline static/src/js/containers/SidebarContainer/Header/Header.scss]: I don't think we need to fix it now, but I am not a huge fan of these relative links

File: static/src/js/components/Header/Header.js (hunk 2 above)
Comment (macrouch) [reply static/src/js/components/Header/Header.js]: :doh:

**Comment by abbottry** [comment]
Looks like by removing the `.gitignore`, the items that were in that file will show up as changed files if we remove them after pulling this branch down.

Examples: 

- .byebug_history
- config/
- db/
- doc/
- tmp/
- vendor/

Should we keep these items in the new `.gitignore` until they are removed from all of our local dev environments?

**Comment by abbottry** [comment]
Are the git submodules for echoforms and timeline remaining in the application?

**Comment by abbottry** [comment]
I'm still seeing the `performance` folder.

**Comment by abbottry** [comment]
Keeping `bin/ecc-sync` for potential reuse?

**Comment by abbottry** [comment]
Can we add `.DS_Store` to the `.gitignore`?

**Pull request merged**